### PR TITLE
Bump controller-runtime to 0.8.0

### PR DESCRIPTION
**Pull request by varshaprasad96**
**Description of the change:**
Bump controller-runtime to 0.8.0

**Motivation for the change:**
To support k8s 1.20

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,9 +8,8 @@ require (
 	github.com/operator-framework/api v0.4.0
```
Comment (varshaprasad96) [inline go.mod:8]: Need to bump this. Waiting on https://github.com/operator-framework/api/pull/94

Comment (estroz) [reply go.mod:8]: This can be updated to v0.5.2

**Review by estroz — APPROVED** [review]
/approve
/lgtm

**Pull request merged**